tests/func/test_stage.py

```diff
@@ -178,3 +178,14 @@ def test_meta_is_preserved(tmp_dir, dvc):
 
     new_data = load_stage_file(stage.path)
     assert new_data["meta"] == data["meta"]
+
+
+def test_parent_repo_collect_stages(tmp_dir, scm, dvc, subrepo_dir):
```

Comment:
Looks like this is the only case where we are using `subrepo_dir` fixture,right?  So maybe let's not mess with current fixtures then, and just create what we need here?